### PR TITLE
[APM] Trace retention update.

### PR DESCRIPTION
**Pull request by nmuesch**

### CODE REVIEW

File: content/tracing/faq.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ customnav: tracingnav
 
 ### How long is tracing data stored?
 
-Tracing data is stored for about 36 hours.
+Traces will be probabilistically available for up to 6 months, unless viewed. If you view a Trace, it is guaranteed to be stored for 6 months.
```
Comment (lejulie) [inline content/tracing/faq.md]: I think "probabilistically available" sounds a little confusing.  Maybe "Traces are stored for up to 6 months; the storage time for each trace is determined probabilistically.  If you view a trace, it is guaranteed to be stored for 6 months."

**Comment by l0k0ms** [comment]
Updated on new doc, We can either merge it with current master or wait for the new doc, @lejulie do you want this published right now?

**Comment by lejulie** [comment]
@l0k0ms how long is the wait for the new doc?  What's up there now isn't right so ideally it should be fixed asap.

**Comment by l0k0ms** [comment]
@lejulie 2 weeks ;) but we are going to merge it on the current doc as well.

**Pull request merged**